src/lti/LTI_Message_Launch.php

```diff
@@ -276,15 +283,14 @@ private function validate_nonce() {
     }
 
     private function validate_registration() {
+        $client_id = is_array($this->jwt['body']['aud']) ? $this->jwt['body']['aud'][0] : $this->jwt['body']['aud'];

```

Comment:
I'm wondering if we should add a helper method to this class to represent this, since we're also doing it in our own code in TARL.